docs/en/writing-running-appium/other/how_to_get_appium_server_logs.md

```diff
@@ -0,0 +1,18 @@
+# How to get Appium server logs
+  If there's an issue with a test execution, the Appium server logs are normally the best way to find the root of the issue. This document shows how to get the Appium server logs and a short explaination of how to read them.
```

Comment:
* Typo in `explaination`... should be `explanation.
* I would just delete the first sentence